app/markets/MarketDetails.tsx

```diff
@@ -65,9 +73,44 @@ export const MarketDetails = ({ id }: MarketDetailsProps) => {
               <OutcomeBar market={market} />
             </div>
           </div>
-          {!marketModel.isClosed && (
+          <div className="px-4">
+            <ToggleGroup
+              value={tab}
+              onChange={setTab}
+              className="w-full md:w-full justify-around"
+            >
+              <div className="w-full">
+                <ToggleGroupOption
+                  size="md"
+                  value={"bet"}
+                  className="font-semibold flex justify-center"
+                >
+                  Bet
+                </ToggleGroupOption>
+              </div>
+              <div className="w-full">
+                <ToggleGroupOption
+                  size="md"
+                  value={"history"}
+                  className="font-semibold flex justify-center"
+                >
+                  History
+                </ToggleGroupOption>
+              </div>
+            </ToggleGroup>
+          </div>
+          {tab === "bet" && (
+            <div className="p-2">
+              {!marketModel.isClosed ? (
+                <Swapbox market={market} />
+              ) : (
+                <div className="p-4 text-center">Market closed</div>
+              )}
+            </div>
+          )}
+          {tab === "history" && (
             <div className="p-2">
-              <Swapbox market={market} />
+              <div className="p-4 text-center">Market history</div>
```

Comment:
```suggestion
              <div className="p-4 text-center">No History yet</div>
```

Reply:
this is just a placeholder at this stage, as we we're building by steps.